langchain4j-bedrock/src/main/java/dev/langchain4j/model/bedrock/BedrockCohereEmbeddingModel.java

```diff
@@ -0,0 +1,78 @@
+package dev.langchain4j.model.bedrock;
+
+import dev.langchain4j.data.segment.TextSegment;
+import dev.langchain4j.model.bedrock.internal.AbstractBedrockEmbeddingModel;
+import lombok.Builder;
+import lombok.Getter;
+import lombok.experimental.SuperBuilder;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+/**
+ * Bedrock Cohere Embedding Model with support for both versions:
+ * {@code cohere.embed-english-v3} and it's multilingual version {@code cohere.embed-multilingual-v3}
+ * <br>
+ * See more details <a href="https://docs.aws.amazon.com/bedrock/latest/userguide/model-parameters-embed.html">here</a>.
+ */
+@SuperBuilder
+@Getter
```

Comment:
It would be great if you could de-lombok, we are trying to get rid of it

Reply:
I can get rid of @Getter (will provide new update on pull request) but @SupperBuilder is used based on that `AbstractBedrockEmbeddingModel.java` and maybe I can work in other PR on this?